client/src/entry/analysis/router.js

```diff
@@ -570,6 +571,14 @@ export function getRouter(Galaxy) {
                             datasetId: route.query.dataset_id,
                         }),
                     },
+                    {
+                        path: "visualizations/create",
+                        component: VisualizationCreate,
+                        name: "VisualizationsCreate",
+                        props: (route) => ({
+                            visualization: route.query.visualization,
+                        }),
```

Comment:
Why did you use a query instead of using a parameter?

```suggestion
                        path: "visualizations/create/:visualization",
                        component: VisualizationCreate,
                        name: "VisualizationsCreate",
                        props: (route) => ({
                            visualization: route.params.visualization,
                        }),
```